TESTS/mbed_hal/common_tickers/main.cpp

```diff
@@ -53,7 +53,7 @@ extern "C" {
 #define TICKER_100_TICKS 100
 #define TICKER_500_TICKS 500
 
-#define MAX_FUNC_EXEC_TIME_US 30
+#define MAX_FUNC_EXEC_TIME_US 20
```

Comment:
@mprse 
Could you kindly elaborate on this? I thought we wanted increase instead of decrease the time taken on execution.

Reply:
Yes, I found this while adding this patch and fixed as a part of it. According to the requirements:

https://github.com/ARMmbed/mbed-os/blob/40b7901cfbdd77fcab99cd04aa62aec24f868fe0/hal/us_ticker_api.h#L91-L92

So this value should be `20 us`. This should not have an impact on `S111` target since it is very fast. I also tested this with a few other targets with positive results.